folium/plugins/fullscreen.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+"""
+Fullscreen
+--------------
+
+https://github.com/brunob/leaflet.fullscreen
+
+Note: does not work from within an iframe, and because of this, does not
+work in jupyter notebook
+"""
```

Comment:
I would put this in the class docstring. Add here just a quick description like `Adds fullscreen button to your maps.`